ls1intum/Artemis
# Reduce bundle size by lazily loading showdown

### Checklist

## Motivation

See https://github.com/ls1intum/Artemis/pull/2743#pullrequestreview-576131019.

Gzip bundle size is `1.45MB` now:
![grafik](https://user-images.githubusercontent.com/72132281/106159621-4ae50600-6185-11eb-8226-fe7e812c3be5.png)


## Steps for Testing

1. Make sure instructions/descriptions for any exercise types still display/render correctly (both for the instructors, and for the students in the details or exam participation views)
2. The same must work in the exam mode, too!
3. Make sure quiz questions still display and render correctly
4. Check if exercise hints, lectures, lecture units and the guided tour still display correctly
5. Student questions and answers need to work as well

**Updated:** 7. Test that the guided tour still works as expected

So tl;dr: Everything using the Markdown editor or Markdown display needs to be tested.
